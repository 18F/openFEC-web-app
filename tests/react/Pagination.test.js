const rendered = require('react-test-renderer');
const React = require('react');
const ReactDOM = require('react-dom');
const Pagination = require('../../static/js/legal/Pagination');

it('renders pagination', () => {
  const component = rendered.create(
    <Pagination from_hit={0} advisory_opinions={[]}
      resultCount={0} handleChange={() => {}} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
