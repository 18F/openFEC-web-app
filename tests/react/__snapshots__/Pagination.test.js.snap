// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders pagination 1`] = `null`;
